test/index.spec.js

```diff
@@ -730,6 +738,44 @@ describe('onBuild()', () => {
     expect(existsSync(path.join('.netlify', 'edge-functions', 'ipx', 'index.ts'))).toBeTruthy()
   })
 
+  test('generates edge-functions manifest', async () => {
+    await moveNextDist()
+    await nextRuntime.onBuild(defaultArgs)
+    expect(existsSync(path.join('.netlify', 'edge-functions', 'manifest.json'))).toBeTruthy()
+  })
+
+  test('generates generator field within the edge-functions manifest', async () => {
+    await moveNextDist()
+    await nextRuntime.onBuild(defaultArgs)
+    const manifestPath = await readJson(path.resolve('.netlify/edge-functions/manifest.json'))
+    const manifest = manifestPath.functions.sort()
```

Comment:
if the functions are sorted now, can we not use snapshots?

Reply:
The sort should have been deleted along with the snapshot, I will make that update.